sound/soc/sof/ipc.c

```diff
@@ -264,6 +264,10 @@ int sof_ipc_tx_message(struct snd_sof_ipc *ipc, u32 header,
 	struct snd_sof_ipc_msg *msg;
 	unsigned long flags;
 
+	/* todo: fix me: temporary disable ipc message sending */
```

Comment:
why is this needed? You didn't initialize IPC in the first patches so would this code even be used?

Reply:
the topology parsing still try to send IPC even neither of dsp and IPC is initialized.
this is hack, let me refine it.